lib/Bio/EnsEMBL/DataCheck/Checks/CheckSpeciesSetSizeByMethod.pm

```diff
@@ -68,16 +69,22 @@ sub tests {
       my $gdbs = $species_set->genome_dbs;
       my $gdb_count = scalar( @$gdbs );
 
-      my $desc = "For MLSS $mlss_name there are $allowable_count genome_dbs for species_set $species_set_name ($species_set_id), as expected";
-      if ( ($method eq 'LASTZ_NET') && ($gdb_count == 1) && ($species_set_name !~ /-/) ) {
+      if ( ($method eq 'LASTZ_NET' || $method eq 'SYNTENY') && ($gdb_count == 1) && ($species_set_name !~ /-/) ) {
         $allowable_count = 1;
       }
+
+      my $desc = "For MLSS $mlss_name there are $allowable_count genome_dbs for species_set $species_set_name ($species_set_id), as expected";
+
       is( $gdb_count, $allowable_count, $desc );
       if ( $mlss_name =~ /(^[0-9]+) / ) {
         is($1, $gdb_count, "species_set $species_set_name and mlss $mlss_name both link to $gdb_count genomes");
       }
+      $found = 1;
     }
   }
+  unless ($found) {
```

Comment:
I chose `unless ($found)` because it's the shortest form. If we go for a longer form, I would prefer a regular `if`, e.g.
```suggestion
  if ($found == 0) {
```

Reply:
I am sure it doesn't matter because if it works it works. I just found when I was going through, to use `unless` like that doesn't seem quite right, because `$found` has been initialised: `my $found = 0;` I prefer the longer form in this case. However, it is not an approval blocker!